src/tool/cppwinrt/cppwinrt/helpers.h

```diff
@@ -62,9 +68,52 @@ namespace xlang
             return name;
         }
 
+        MethodDef const& method() const
+        {
+            return m_method;
+        }
+
+        bool is_async() const
+        {
+            if (is_put_overload(m_method))
```

Comment:
per the detailed description above, this function's body is optimized for actual usage in a specific case, rather than simply providing an answer to whether the method is async. It would make sense to pick a more specific name and/or add a comment that's a bit more durable. Otherwise it's very non-obvious as to why a property setter is considered "async".

Reply:
Sure, I'll add some of that explanation as a comment.

Reply:
Done - added a comment.